content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,537 @@
+Alpha is the level we set our p-value at inorder to ascertain whether to accept our null hypothesis or not.
+Our alpha value is the probability of making a wrong decision when the ground truth(H0) is true. It is always calculated as
+```
+alpha = 1 - confidence level
+```
+If the confidence level from what we've seen in our data is 95%, then
+```
+alpha = 1 - 0.95
+```
+which is 0.05.
+ 
+Why is the 95% confidence level the most accustomed?
+ 
+Since we know that alpha level is the probability of encountering a Type I error(which is rejecting H0 despite being true), would it not make sense to make it as bitsy as possible. Imagine our alpha value being set to 10%, obviously, there is an increasing chance that we will make the mistake of rejecting the null hypothesis, but it must not be too small as well, if we set the alpha value as 1%, it will be too small that our decision will be reeking of bias. Hence, the standard has been 5%.
+ 
+While implementing statistical hypothesis testing, there are some methodologies that can be employed. In this tutorial, I will be touching on three methods of carrying out statistical hypothesis testing.
+ 
+ 
+### Statistical Techniques for Hypothesis Testing
+The three methods to be discussed are:
+1. Chi-Square Test
+2. T-Test
+3. ANOVA Test
+ 
+#### Chi-Square Test
+ 
+Chi-square test is used to perform testing on two categorical variables in our data population. It is focused on looking for an important relationship between two categorical variables.
+For example, comparing the aftereffects of malaria drug A, malaria drug B, and malaria drug C.
+Chi-square is a tool for checking how divergent the observed frequency is from the expected frequency.
+With Chi-Square, we accept the null hypothesis when variable A and variable B are not dependent, i.e. there is no relationship between them.
+We reject the null hypothesis(or accept the alternate hypothesis) when variable A depends on variable B, i.e. there is a relationship between them. The formula for calculating chi-square test is:
+ 
+![chi_square_formula](/engineering-education/statistical-hypothesis-testing-python-implementation/chi-square.PNG)
+ 
+Consider the following example:
+ 
+A mathematics teacher conducted a test for a class of 20 students. He expected 5 to score good marks, 7 to score average marks while 8 to fail. Eventually, when the result came out, 5 failed, 6 were average while 9 scored good marks. Mathematically, carry out chi-square test for this class.
+ 
+|  | Good  |  Average  | Fail | Total |
+|----|--------|:----------|--------------|--------|
+| Observed| 9 | 6 | 5 | 20 |
+| Expected| 5 | 7 | 8 | 20 |
+ 
+Where
+ 
+- observed good is A,
+- expected good is a,
+- observed average is B,
+- expected average is b,
+- observed fail is C, and
+- expected to fail is c.
+```
+Chi-square = Σ[(observed value - expected value)² / expected value]
+```
+```
+A-a = 4
+B-b = -1
+C-c = -3
+```
+```
+# find the square of our values
```

Comment:
Either remove this line or do put it outside the formatting block without the `#`